rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
```

Comment:
I would also be tempted to move this line into the "`playgame`" function. Although it's not relevant at this stage, if you wanted to use the `playgame` function in another program you couldn't import it without this message showing up.